src/libs/getCurrentPosition/index.desktop.ts

```diff
@@ -0,0 +1,46 @@
+import type {GetCurrentPosition} from './getCurrentPosition.types';
+import {GeolocationErrorCode} from './getCurrentPosition.types';
+
+const makeError = (code: (typeof GeolocationErrorCode)[keyof typeof GeolocationErrorCode], message: string) => ({
```

Comment:
Use `ValueOf` from `type-fest` for improved readability:

```suggestion
const makeError = (code: ValueOf<typeof GeolocationErrorCode>, message: string) => ({
```